hypha/apply/activity/adapters/slack.py

```diff
@@ -372,7 +373,10 @@ def send_message(self, message, recipient, source, **kwargs):
         for room in target_rooms:
             try:
                 slack_message('messages/slack_message.html', data, channel=room)
+            except ChannelNotFound as e:
+                logger.warning(e)
+                return '400: Bad Request'
```

Comment:
Won't this still lead to an error? whats' the difference b/w the `ChannelNotFound` and the general `Exception` handler, both return 400. Logger won't halt the execution anyway.

Reply:
@theskumar There wasn't any UI error or execution halt for the user. It is working in the same way as before, we just avoid the sentry noise. Previously we were using the logger.exception and it was hitting sentry, so for ChannelNotFound we just use warning to avoid sentry and error for other exceptions.
And I think returning and logging error is fine here until we implement the 'check channel existence by sending a message' feature. And as this is not a UI error, just logging, so not an issue i think.